NumLib/Fem/ShapeFunction/ShapePyra13.h

```diff
@@ -28,7 +28,7 @@ class ShapePyra13
      * @param [out] N   a vector of calculated shape functions
      */
     template <class T_X, class T_N>
-    static void computeShapeFunction(const T_X &r, T_N &N);
+    static void computeShapeFunction(const T_X& x, T_N& N);
```

Comment:
Is the substitution of `r` by `x` intended? Then the docu have to be changed, too.

Reply:
@bilke Why didn't doxygen fail?
@TomFischer Yes, intended. clang-tidy is "insisting" on same argument names in declaration and definition ;) I'll change the docs of course...